docs/flowcollector-flows-netobserv-io-v1alpha1.adoc

```diff
@@ -0,0 +1,1347 @@
+     // other fields }
+
+| `namespace`
+| `string`
+| namespace where console plugin and flowlogs-pipeline have been deployed.
+
+|===
+== .status.conditions
+Description::
++
+--
+conditions represent the latest available observations of an object's state
+--
+
+Type::
+  `array`
+
+
+
+
+== .status.conditions[]
+Description::
++
+--
+Condition contains details for one aspect of the current state of this API Resource. --- This struct is intended for direct use as an array at the field path .status.conditions.  For example, type FooStatus struct{     // Represents the observations of a foo's current state.     // Known .status.conditions.type are: "Available", "Progressing", and "Degraded"     // +patchMergeKey=type     // +patchStrategy=merge     // +listType=map     // +listMapKey=type     Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"` 
+     // other fields }
+--
+
+Type::
+  `object`
+
+Required::
+  - `lastTransitionTime`
+  - `message`
+  - `reason`
+  - `status`
+  - `type`
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `lastTransitionTime`
+| `string`
+| lastTransitionTime is the last time the condition transitioned from one status to another. This should be when the underlying condition changed.  If that is not known, then using the time when the API field changed is acceptable.
+
+| `message`
+| `string`
+| message is a human readable message indicating details about the transition. This may be an empty string.
+
+| `observedGeneration`
+| `integer`
+| observedGeneration represents the .metadata.generation that the condition was set based upon. For instance, if .metadata.generation is currently 12, but the .status.conditions[x].observedGeneration is 9, the condition is out of date with respect to the current state of the instance.
+
+| `reason`
+| `string`
+| reason contains a programmatic identifier indicating the reason for the condition's last transition. Producers of specific condition types may define expected values and meanings for this field, and whether the values are considered a guaranteed API. The value should be a CamelCase string. This field may not be empty.
+
+| `status`
+| `string`
+| status of the condition, one of True, False, Unknown.
+
+| `type`
+| `string`
+| type of condition in CamelCase or in foo.example.com/CamelCase. --- Many .condition.type values are consistent across resources like Available, but because arbitrary conditions can be useful (see .node.status.conditions), the ability to deconflict is important. The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
```

Comment:
```suggestion
| type of condition in CamelCase or in foo.example.com/CamelCase. --- Many .condition.type values are consistent across resources like Available, but because arbitrary conditions can be useful (see .node.status.conditions), the ability to deescalate is important. The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
```

Reply:
The [Vale](https://vale.vercel.app/) linter is saying this isn't a word so can we say "deescalate"